docs/hdfs.md

```diff
@@ -21,6 +21,10 @@ For more information, see [Inheriting Hadoop Cluster Configuration][8].
 
 For DC/OS HDFS, these configuration files are served at `http://<hdfs.framework-name>.marathon.mesos:<port>/v1/connection`, where `<hdfs.framework-name>` is a configuration variable set in the HDFS package, and `<port>` is the port of its marathon app.
 
+### Spark Checkpointing
+
+In order to use spark with cehckpointing make sure you follow the instructions [here](https://spark.apache.org/docs/latest/streaming-programming-guide.html#checkpointing) and use an hdfs directory as the checkpoint directory. In the future we plan to support checkpointing at the driver level with local persistent volumes.
```

Comment:
@skonto This is great. Could you add an example? Also, please remove `In the future we plan to support checkpointing at the driver level with local persistent volumes.` although I like the optimism, let's put this in _when_ we have driver level checkpointing.